src/schema/trace.ts

```diff
@@ -16,25 +16,25 @@ export function traceResolver<TSource, TArgs, TReturn>(
     context: Context,
     info: GraphQLResolveInfo,
   ): Promise<TReturn> => {
-    if (context?.span) {
-      const name = `${info.parentType.name}.${info.fieldName}`;
-      const childSpan = context.span.createChildSpan({ name });
-      childSpan.addLabel('/graphql/parent', info.parentType.name);
-      childSpan.addLabel('/graphql/field', info.fieldName);
-      childSpan.addLabel(
-        '/graphql/operation/name',
-        info.operation?.name?.value,
-      );
-      childSpan.addLabel('/graphql/operation/type', info.operation?.operation);
-      try {
-        const res = await next(source, args, context, info);
-        childSpan.endSpan();
-        return res;
-      } catch (err) {
-        childSpan.endSpan();
-        throw err;
-      }
-    }
+    // if (context?.span) {
```

Comment:
I guess we don't need it anymore at all right?

Reply:
Probably not, just kept it there for reference if I might still use it. Will remove it when I do a cleanup.

Reply:
re-introduced this, using it for metrics and adding attributes to the root span